languages/python/exercises/concept/dicts/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+In this exercise you'll be managing an inventory system.
```

Comment:
Nice story! You could add a little bit of flair to the story by possibly adding what type of items the inventory system will hold. Like a "theme" of sorts.

Reply:
Ah okay!
Something I will add.